src/main/java/org/candlepin/subscriptions/resource/InstancesResource.java

```diff
@@ -208,6 +215,9 @@ private InstanceData asTallyHostViewApiInstance(
           Optional.ofNullable(host.getMonthlyTotal(monthId, Measurement.Uom.fromValue(uom)))
               .orElse(0.0));
     }
+    if (!host.getBuckets().isEmpty()) {
+      instance.setCategory(host.getBuckets().iterator().next().getMeasurementType().toString());
+    }
```

Comment:
I'd prefer:

```suggestion
    host.getBuckets().stream()
        .findFirst()
        .map(HostTallyBucket::getMeasurementType)
        .map(Objects::toString)
        .ifPresent(instance::setCategory);
```
It's just a slight bit more idiomatic.